src/context/StoreContext.js

```diff
@@ -56,23 +58,42 @@ export default function StoreContextProvider(props) {
     null
   )
 
+  const onBibleRefUIChange = (bId, ch, v) => {
+    console.log(`\n### Reference changed to ${bId} - ${ch}:${v}\n\n`);
```

Comment:
Could you remove this `console.log` please? It clutters the console for other developers

Reply:
Done